spec/spec.md

```diff
@@ -1 +1,298 @@
+
+The LinkedClaims specification is being developed alongside several complementary initiatives:
+
+* [Progressive Trust](TBD) - A framework for building trust through incremental verification and validation of claims
+* [digest Multibase Hashlink](TBD) - TBD
+* [inbox](TBD) - The use of an ActivityPub compatible inbox for replies to claims
+
+## Status of This Document
+
+Decentralized Web Node is a _DRAFT_ specification under development within the [Decentralized Identity Foundation](https://identity.foundation) (DIF). It is an active work item of the [Linked Claim Incubation Lab at DIF](https://github.com/decentralized-identity/labs/blob/main/proposals/linked_claims/001_proposal.md)  It incorporates requirements and learnings from diverse stakeholders across sectors into a shared specification that meets the collective needs of the community.
+
+The specification will be updated to incorporate feedback, from DIF members and the wider community, with a reference implementation being developed that exercises the features and requirements defined here. We encourage reviewers to submit [GitHub Issues](https://github.com/decentralized-identity/labs-linkedclaims/issues) as the means by which to communicate feedback and contributions.
+
+## Terminology
+
+[[def:DID]]
+~ A Decentralized Identifier as defined in https://www.w3.org/TR/did-core/
+
+[[def:Linked Claim]]
+~ A structured, cryptographically signed document with a URI subject that is itself addressable at a URI, following at least the MUST requirements detailed below.
+
+[[def:Signer]]
+~ The entity that cryptographically signs a claim (known as the "issuer" in Verifiable Credentials)
+
+[[def:URI]]
+~ a Uniform Resource Identifier as defined in https://datatracker.ietf.org/doc/html/rfc3986
+
+## Core Pattern
+
+```mermaid
+
+graph LR
+    subgraph Claim-B
+        B_id["id: URI_B"]
+        B_subject["subject: URI_A"]
+        B_content["content: {...}"]
+        B_sig["signature: xyz"]
+    end
+    
+    subgraph Claim-A
+        A_id["id: URI_A"]
+        A_subject["subject: URL_X"]
+        A_content["content: {...}"]
+        A_sig["signature: xyz"]
+    end
+
+    subgraph Resource
+        X["URL_X"]
+    end
+
+    %% Claims point to their subjects
+    B_subject -.-> A_id
+    A_subject -.-> X
+    
+    classDef claim fill:#f8f8ff,stroke:#333,stroke-width:2px
+    classDef claimPart fill:#ffffff,stroke:#666,stroke-width:1px
+    classDef resource fill:#e8f4f8,stroke:#333,stroke-width:2px
+    
+    class B_id,B_subject,B_content,B_sig,A_id,A_subject,A_content,A_sig claimPart
+    style Claim-A fill:#f8f8ff,stroke:#333,stroke-width:2px
+    style Claim-B fill:#f8f8ff,stroke:#333,stroke-width:2px
+    style Resource fill:#e8f4f8,stroke:#333,stroke-width:2px
+    style X fill:#ffffff
+    
+    linkStyle 0,1 stroke:#666,stroke-width:2px,stroke-dasharray: 5
+```
+
+## LinkedClaim Conformance Requirements
+
+A LinkedClaim:
+
+* **MUST** have a subject that can be any valid URI
```

Comment:
These need to link to proper specifications. Valid, use the RFC's in IETF. I think I shared them.

Reply:
i thought i put that up in terminology, i can also make them a link.

Reply:
added link